bdnex/lib/bdgest.py

```diff
@@ -397,12 +419,27 @@ def parse_album_metadata_mobile(self, album_name):
         if isinstance(album_meta_dict['Planches'], str):
             album_meta_dict['Planches'] = int(album_meta_dict['Planches'])
 
-        if hasattr(album_meta_dict, 'Tome'):
+        if 'Tome' in album_meta_dict.keys():
```

Comment:
@jpmenil , sorry for the messy PR doing too many things in one. This is similar to https://github.com/lbesnard/bdnex/pull/13

Reply:
:)